### ISSUE TITLE
Enable GitHub Discussions

### ISSUE THREAD

**Issue by filip26** [w3c/vc-data-integrity#279]
GitHub discussions provide more features than plain comments below an issue and it could be a great place to answer questions or/and having (long) discussions over complicated topics. See some comments added to issue #272 that are valid questions that should be answered but are just loosely related to the issue. 

In the best case scenarios, discussions - questions/answers could become another source of information helping implementers and developers to get aboard.

**Comment by aljones15** [comment]
More information on [Github Discussions here.](https://github.com/features/discussions) I am +1 to this, but we might consider using discussions on specific repos first before making it universal. Data Integrity would be a good start.

**Comment by filip26** [comment]
I understand that to make such a decision might involve a deep conversation and must fit with internal W3C workflow and policies. I'm just not sure that all those listed channels could be seen as a replacement. GitHub discussions are something like an alternative to SO focused on an integration with a repo - I'm not advocating for GitHub, but for something better than maillist + issues + irc (omitting calls - not the right place to ask for help or have long complicated discussions) . 

Just, here is a link on EUDI ARF GitHub repo, as an example.
https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/discussions

**Comment by msporny** [comment]
@filip26 Yes, #272 should be a discussion, just like https://github.com/w3c/vc-data-model/issues/947 and https://github.com/w3c/vc-data-model/issues/929 should have been a "discussion".

That said, this directly impacts me and the paperwork I have to do to ensure that the WG hasn't left itself open to any formal objections. Opening up yet another place where we have to prove the WG processed a "discussion" appropriately, followed W3C Process, can point to WG call log discussion, was labeled appropriately, went through the objection/proposal/resolution stages... none of that is worked out for Github discussions (or any other thing).

The way W3C Process works is that to get a global standard, the WG has to prove that it processed every single comment against the specification per W3C Process... not doing so, even for a single issue, can result in a formal objection. Anyone on the planet can formally object to a proposal -- W3C specifications have an extremely high bar for consensus (and many of us view this as a good thing). The downside is, of course, that we have to be able to prove that we've met that high bar... and by "we", I mostly mean a handful of largely overworked/volunteer Editors, Chairs, and Staff Contacts. I know you understand what I'm getting at considering you're a well regarded and prolific open source developer. :)

We will discuss this with the WG (because we are compelled by W3C Process to do so, an issue was raised afterall), though I would rather we don't open up Github discussions as it can be used as an attack vector against us finishing the work: "Oh, you didn't resolve that discussion! I formally object! I said 10 things in that discussion that the WG didn't consider!"... etc.
